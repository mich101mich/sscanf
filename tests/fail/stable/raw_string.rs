fn main() {
    sscanf::sscanf!("hi", r"asdf{}asdf");
    sscanf::sscanf!("hi", r"asdf{:bob}asdf", usize);
    sscanf::sscanf!("hi", r"asdf{usize:bob}asdf");
    sscanf::sscanf!("hi", r##"asdf{}asdf"##);
    sscanf::sscanf!("hi", r##"asdf{:bob}asdf"##, usize);
    sscanf::sscanf!("hi", r##"asdf{usize:bob}asdf"##);
}
