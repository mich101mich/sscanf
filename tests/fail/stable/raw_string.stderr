error: more placeholders than types provided:
       At r"asdf{}asdf"
                ^^
 --> tests/fail/stable/raw_string.rs:2:27
  |
2 |     sscanf::sscanf!("hi", r"asdf{}asdf");
  |                           ^^^^^^^^^^^^^

error: unrecognized format option.
       Hint: Regex format options must start and end with '/':
       At r"asdf{:bob}asdf"
                  ^^^
 --> tests/fail/stable/raw_string.rs:3:27
  |
3 |     sscanf::sscanf!("hi", r"asdf{:bob}asdf", usize);
  |                           ^^^^^^^^^^^^^^^^^

error: unrecognized format option.
       Hint: Regex format options must start and end with '/':
       At r"asdf{usize:bob}asdf"
                       ^^^
 --> tests/fail/stable/raw_string.rs:4:27
  |
4 |     sscanf::sscanf!("hi", r"asdf{usize:bob}asdf");
  |                           ^^^^^^^^^^^^^^^^^^^^^^

error: more placeholders than types provided:
       At r##"asdf{}asdf"##
                  ^^
 --> tests/fail/stable/raw_string.rs:5:27
  |
5 |     sscanf::sscanf!("hi", r##"asdf{}asdf"##);
  |                           ^^^^^^^^^^^^^^^^^

error: unrecognized format option.
       Hint: Regex format options must start and end with '/':
       At r##"asdf{:bob}asdf"##
                    ^^^
 --> tests/fail/stable/raw_string.rs:6:27
  |
6 |     sscanf::sscanf!("hi", r##"asdf{:bob}asdf"##, usize);
  |                           ^^^^^^^^^^^^^^^^^^^^^

error: unrecognized format option.
       Hint: Regex format options must start and end with '/':
       At r##"asdf{usize:bob}asdf"##
                         ^^^
 --> tests/fail/stable/raw_string.rs:7:27
  |
7 |     sscanf::sscanf!("hi", r##"asdf{usize:bob}asdf"##);
  |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
